app/hub/Views/UpgradePlanView/UpgradePlanView.jsx

```diff
@@ -419,13 +419,13 @@ const UpgradePlanView = (props) => {
 				<div className="mobile-table-header">
 					<div className="row align-top align-center">
 						<div className="small-4 text-center columns">
-							<p className="protection-description blue">{t('ghostery_basic')}</p>
+							<p className="protection-description blue">Basic</p>
```

Comment:
As per Teresa's list in the translations ticket, "Basic" should be translated, no? (Not "Plus" or "Premium", though)

Reply:
Gotcha, should have double checked that--my bad!

Reply:
Yes that's correct! Good catch